### ISSUE TITLE
Add spear-mtb to jasen

### ISSUE THREAD

**Issue by ryanjameskennedy** [genomic-medicine-sweden/jasen#203]
A lot of work has already gone into tb by Karolinka Institute. Here you'll find the [pipeline](https://github.com/xrazmo/spear-mtb).

**Comment by mhkc** [comment]
How would you like to incorporate this? It the spear-mtb pipeline seem to be similar to JASEN tb options so I am not sure that we need to fork this pipeline.

**Comment by ryanjameskennedy** [comment]
I don't believe that there is much to add from their pipeline to ours, closing...
